pybamm/models/full_battery_models/lithium_metal/dfn.py

```diff
@@ -2,7 +2,7 @@
 # Doyle-Fuller-Newman (DFN) Model
 #
 import pybamm
-from ..lithium_ion.dfn import DFN as LithiumIonDFN
+from pybamm.models.full_battery_models.lithium_ion.dfn import DFN as LithiumIonDFN
```

Comment:
Doing this across the codebase would be much appreciated, though in a separate PR – thanks!

Reply:
Yes, that is the plan. We had a discussion about this the other day in the `#infrastructure` channel on Slack, I shared [`mkinit`](https://pypi.org/project/mkinit/) for this to do this as a one-time thing, but additionally, a pre-commit hook that checks/converts relative imports would be great as well. 

I found https://github.com/MarcoGorelli/absolufy-imports from this [Reddit post](https://www.reddit.com/r/learnpython/comments/1d3n991/is_there_a_python_precommit_tool_that_can_change/), and while it's archived, it might still work

Reply:
https://docs.astral.sh/ruff/rules/relative-imports/

Reply:
Ruff has it, no new tools needed

Reply:
Awesome, thanks!

Reply:
Opened #4134 for this, can be further assigned and discuss there.
Marking this one as resolved for now.